pkg/kv/fs.go

```diff
@@ -0,0 +1,67 @@
+package kv
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"sync"
+
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+type FSStore struct {
+	lock   *sync.RWMutex
+	fsPath string
+}
+
+func NewFSStore(logger *zap.Logger, fsPath string) *FSStore {
+	return &FSStore{
+		lock:   new(sync.RWMutex),
+		fsPath: fsPath,
+	}
+}
+
+func (s *FSStore) Start(ctx context.Context, g *errgroup.Group) error {
+	return nil
+}
+func (s *FSStore) Get(ctx context.Context, ns Namespace, key string) (string, error) {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+
+	path := fmt.Sprintf("%s/%s/%s", s.fsPath, ns, key)
```

Comment:
We won't join file path like this because different platforms use different separator, e.g. Windows use `\`, mac and linux use `/`. 

Please use https://pkg.go.dev/path#Join . It is os-aware